frontend/src/metabase/common/components/Table/Table.unit.spec.tsx

```diff
@@ -72,7 +72,7 @@ const renderRow = (row: Pokemon) => {
 
 describe("common > components > Table", () => {
   it("should render table headings", () => {
-    render(
+    renderWithProviders(
```

Comment:
why does this need providers now? A table component should not depend on the redux store

Reply:
Good point. I've changed the component so it no longer invokes the `useLocale()` hook and no longer depends on the Redux store